packages/cms/form/select/select.tsx

```diff
@@ -1,31 +1,35 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { FormComponentProps } from '@tensei/components'
-import { EuiSelect } from '@tensei/eui/lib/components/form'
+import { EuiSuperSelect } from '@tensei/eui/lib/components/form'
 
 const FormSelect: React.FC<FormComponentProps> = ({
   field,
   name,
   id,
-  value,
-  onChange,
   onFocus,
   error
 }) => {
-  const options = field?.selectOptions?.map(option => {
-    return { text: option.label, value: option.value }
-  })
+  const options =
+    field?.selectOptions?.map(option => {
+      return { value: option.label, inputDisplay: option.value }
```

Comment:
I think you did the reverse. The value should be `option.value` and `inputDisplay` should be `option.label`.